### DISCUSSION TITLE
Microsoft exchange CU and SP notification for server clients

### DISCUSSION THREAD

**Discussion by multispeed** [amidaware/tacticalrmm#548] [Ideas]
Hi,

I was thinking of a feature were the rmm can notify a server installation when there is a newer CU or SP than the installed version, since these installations don't come through windows update. And this is something you can quickly forget if there is not a notifying patch policy in a company.

If you look at the software tab in the rmm installation the exchange server installations are already present with the version number.
![image](https://user-images.githubusercontent.com/22557639/120167822-7c86e000-c1fe-11eb-9de3-48bfdc4cbe3b.png)

Maybe it's an option to match them against this page https://docs.microsoft.com/en-us/exchange/new-features/build-numbers-and-release-dates?view=exchserver-2019 which is maintained and update by microsoft. https://docs.microsoft.com/en-us/exchange/new-features/build-numbers-and-release-dates?view=exchserver-2019

**Comment by bbrendon** [comment]
Sounds like a good powershell script.

**Comment by wh1te909** [comment]
yes this kind of feature is way too specific to be built into the rmm, we can add it as a community script though. we have a very powerful scripting engine this would be great for a script check or automated task

**Reply by multispeed** [reply]
> yes this kind of feature is way too specific to be built into the rmm, we can add it as a community script though. we have a very powerful scripting engine this would be great for a script check or automated task

Sounds great, but will a powershell be able to see if there is a new CU or service pack available

**Reply by dinger1986** [reply]
If there's a way to monitor it via powershell then yes it will work fine. 

If you can find the commands or a script to check updates via powershell (or anything else) there's plenty of guys on discord who can help. 

If powershell can't monitor it or you can't find a script already out there, without that there's not going to be a magic way to do it
